src/fluent_api/select_builder.rs

```diff
@@ -744,6 +748,8 @@ where
         }
     }
 
+    // Specify the listener
+    // TODO: what does the `FirestoreListenerTarget` do?
```

Comment:
`FirestoreListenerTarget` - not my invention :)

https://firebase.google.com/docs/firestore/reference/rpc/google.firestore.v1#google.firestore.v1.Target